project/Build.scala

```diff
@@ -83,9 +83,9 @@ object DottyJSPlugin extends AutoPlugin {
 object Build {
   import ScaladocConfigs._
 
-  val referenceVersion = "3.3.1"
+  val referenceVersion = "3.4.0-RC1"
 
-  val baseVersion = "3.4.0-RC1"
+  val baseVersion = "3.4.1-RC1"
```

Comment:
We should add some tests that check that the minor version N (3.N.X) is equal to the minor TASTy version minus one.